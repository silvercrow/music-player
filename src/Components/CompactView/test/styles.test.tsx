import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import {
  ExtendButton,
  Content,
  ArtCover,
  InfoWrapper,
  Title,
  Subtitle,
  ButtonsWrapper,
  StopButton,
  Arrow,
} from '../styles';

test('ExtendButton', () => {
  const tree = renderer.create(<ExtendButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('Content', () => {
  const tree = renderer.create(<Content />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('ArtCover', () => {
  const tree = renderer.create(<ArtCover />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('InfoWrapper', () => {
  const tree = renderer.create(<InfoWrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('Title', () => {
  const tree = renderer.create(<Title />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('Subtitle', () => {
  const tree = renderer.create(<Subtitle />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('ButtonsWrapper', () => {
  const tree = renderer.create(<ButtonsWrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('StopButton', () => {
  const tree = renderer.create(<StopButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('Arrow', () => {
  const tree = renderer.create(<Arrow />).toJSON();
  expect(tree).toMatchSnapshot();
});
