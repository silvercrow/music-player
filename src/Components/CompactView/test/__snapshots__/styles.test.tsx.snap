// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow 1`] = `
.c0 {
  position: absolute;
  left: 12%;
  top: 5px;
  height: 30px;
  width: 30px;
  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' stroke='%23fff' strokeLinecap='round' strokeLinejoin='round' strokeWidth='2' class='feather feather-chevron-up' width='' viewBox='0 0 24 24'%3E%3Cdefs/%3E%3Cpath d='M18 15l-6-6-6 6'/%3E%3C/svg%3E");
}

<span
  className="c0"
/>
`;

exports[`ArtCover 1`] = `
.c0 img {
  border-radius: 10px;
  width: 32px;
  height: 32px;
  -o-object-fit: cover;
  object-fit: cover;
  margin-right: 16px;
}

<div
  className="c0"
/>
`;

exports[`ButtonsWrapper 1`] = `
.c0 {
  margin-left: auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0"
/>
`;

exports[`Content 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;

exports[`ExtendButton 1`] = `
.c0 {
  position: absolute;
  left: 42%;
  top: -20px;
  padding: 12px 0;
  border-radius: 50%;
  background-color: #5b6467;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  width: 40px;
  height: 40px;
  background-repeat: no-repeat;
  background-size: 20px;
  background-position: center;
  background-color: #ffcd00;
}

.c0:hover {
  top: -30px;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
}

<button
  className="c0"
/>
`;

exports[`InfoWrapper 1`] = `
<div
  className=""
/>
`;

exports[`StopButton 1`] = `
.c0 {
  background-color: #ffff;
  height: 15px;
  width: 15px;
  cursor: pointer;
  margin-right: 24px;
}

<button
  className="c0"
/>
`;

exports[`Subtitle 1`] = `
.c0 {
  font-size: 10px;
  color: #fff;
}

<div
  className="c0"
/>
`;

exports[`Title 1`] = `
.c0 {
  font-size: 12px;
  line-height: 18px;
  font-weight: 800;
  color: #fff;
}

<div
  className="c0"
/>
`;
